rules/false_positives/filebeat.yara

```diff
@@ -0,0 +1,12 @@
+rule misp_mdjson : override {
+  meta:
+    description = "misp_sample.mdjson.log"
+    lvt = "medium"
+  strings:
+    $attribute = "Attribute"
+    $event = "Event"
+    $galaxy = "Galaxy"
+    $shadow = "ShadowAttribute"
+  condition:
+    all of them
```

Comment:
can we add some kind of filesize guardrails on this? It also lowers the scanning load for files that are outside of those limits.